File: daySeven/phoneBook/python/phone_book_main.py
import phone_book

def main_menu():
    while True:
        print("\n===================================================")
        print("\n Welcome to PhoneBook")
        print("1. Add Contact")
        print("2. Remove Contact")
        print("3. Find contact by phone number")
        print("4. Find contact by first name")
        print("5. Find contact by last name")
        print("6. Edit contact")
        print("7. Exit")
        print("===================================================")

        try:
            user_input = int(input("Enter your choice: ").strip())
        except ValueError:
            print("Invalid input. Please enter a number between 1-7.")
            continue

        if user_input == 1:
            phone_book.add_contact()
        elif user_input == 2:
            phone_book.remove_contact()
        elif user_input == 3:
            phone_book.find_contact_by_phone_number()
        elif user_input == 4:
            phone_book.find_contact_by_first_name()
        elif user_input == 5:
            phone_book.find_contact_by_last_name()
        elif user_input == 6:
            phone_book.edit_contact()
        elif user_input == 7:
            print("GoodBye!! Have a nice day!")
            break
        else:
            print("Invalid Option. Please try again.")

if __name__ == "__main__":
    main_menu()

File: daySeven/phoneBook/Java/PhoneBookMain.java
// PhoneBookMain.java

import java.util.Scanner;
public class PhoneBookMain {
     static Scanner input = new Scanner(System.in);
    
public static void main(String[] args) {
int userInput;

	do {
		System.out.println("\n Welcome to PhoneBook");
		System.out.println("1. Add Contact");
		System.out.println("2. Remove Contact");
		System.out.println("3. Find contact by phone number");
		System.out.println("4. Find contact by first name");
		System.out.println("5. Find contact by last name");
		System.out.println("6. Edit contact");
		System.out.println("7. Exit");
		userInput = input.nextInt();
		input.nextLine();
		
		switch(userInput) {
		case 1 -> PhoneBook.addContact();
		case 2 -> PhoneBook.removeContact();
           case 3 -> PhoneBook.findContactByPhoneNumber();
           case 4 -> PhoneBook.findContactByFirstName();
           case 5 -> PhoneBook.findContactByLastName();
           case 6 -> PhoneBook.editContact();
           case 7 -> System.out.println("GoodBye!! Have a nice day!");
           default -> System.out.println("Invalid Option. Please try again "); 
           }
           } 
           while (userInput != 7); 
           }
		
		
}		
